src/RoslynAnalyzers/PublicApiAnalyzers/PublicApiAnalyzers.Help.md

```diff
@@ -13,7 +13,7 @@ To get started with the Public API Analyzer:
 dotnet_public_api_analyzer.require_api_files = true
 ```
 
-See [Analyzer Configuration.md](../../docs/Analyzer%20Configuration.md) for more details on how to setup editorconfig based configuration.
+See [Analyzer Configuration.md](https://github.com/dotnet/roslyn-analyzers/blob/main/docs/Analyzer%20Configuration.md) for more details on how to setup editorconfig based configuration.
```

Comment:
The documentation we were linking to didn't seem terribly relevant to the PublicApiAnalyzers. Perhaps it would be better to link to the MS docs on configuring analysis.

```suggestion
See [Configuration options for code analysis](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/configuration-options) for more details on how to setup editorconfig based configuration.
```